Plugins/PDCPlugin/PDCPlugin.swift

```diff
@@ -307,9 +313,47 @@ struct ModuleBuildRequest {
 
         // MARK: - CLI
 
+        @Sendable func ccURL() throws -> URL {
+            guard let url = [
+                "/usr/local/playdate/gcc-arm-none-eabi-9-2019-q4-major/bin/arm-none-eabi-gcc",
+                try? context.tool(named: "arm-none-eabi-gcc").path.string
+            ].compactMap({ $0 }).filter({
+                FileManager.default.fileExists(atPath: $0)
+            }).map({ URL(filePath: $0) }).first else {
+                Diagnostics.warning("arm-none-eabi-gcc not found. Ensure it is installed and in the PATH.")
+                throw Error.armNoneEabiGCCNotFound
+            }
+            return url
+        }
+
+        @Sendable func clangURL() throws -> URL {
+            guard let url = [
+                try? context.tool(named: "clang").path.string
```

Comment:
Without this I get errors when building:

```suggestion
                swiftToolchain.path + "/usr/bin/clang",
                try? context.tool(named: "clang").path.string
```